packages/explat-client-react-helpers/src/test/index.tsx

```diff
@@ -215,17 +199,14 @@ describe( 'ProvideExperimentData', () => {
 				) }
 			</ProvideExperimentData>
 		);
-		expect( capture.mock.calls.length ).toBe( 1 );
-		expect( capture.mock.calls[ capture.mock.calls.length - 1 ] ).toEqual( [ true, null ] );
+		expect( capture ).toHaveBeenCalledTimes( 1 );
+		expect( capture.mock.calls[ 0 ] ).toEqual( [ true, null ] );
```

Comment:
And here you can say `expect( capture ).toHaveBeenCalledWith( true, null )`.

In short, directly looking at `.mock.calls` is for rare special cases -- most common ones have a convenient wrapper.